sdk/videoanalyzer/azure-media-videoanalyzer-edge/README.md

```diff
@@ -166,6 +166,9 @@ MethodResult setPipelineResult = invokeDirectMethodHelper(dClient, setPipelineTo
 
 ## Troubleshooting
 
+When sending a method request using the IoT Hub's `invoke` remember to not type in the method request name directly. Instead use `MethodRequestName.getMethodName()`
```

Comment:
This doesn't look like other other SDK's troubleshooting, like [app config](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/appconfiguration/azure-data-appconfiguration#troubleshooting).